recipes/cmec_driver/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmec_driver" %}
+{% set version = "1.0.0a5" %}
```

Comment:
Since this is an alpha version, you will probably not want to put release this to the `main` label.  If you do, users may install it by mistake even after v1.0.0 is released.  It would be good to have this go to a `conda-forge/label/cmec_dev` label instead of `main` to prevent this.  To do this, create a `conda_build_config.yaml` file next to `meta.yaml` and put this in it:
```
channel_targets:
  - conda-forge cmec_dev
```